src/libs/ReportUtils.ts

```diff
@@ -3333,18 +3157,17 @@ function chatIncludesChronos(report) {
  * - It was written by someone else and isn't a whisper
  * - It's a welcome message whisper
  * - It's an ADDCOMMENT that is not an attachment
- *
- * @param {Object} reportAction
- * @param {number} reportID
- * @returns {Boolean}
  */
-function canFlagReportAction(reportAction, reportID) {
+function canFlagReportAction(reportAction: OnyxEntry<ReportAction>, reportID: string | undefined): boolean {
```

Comment:
```suggestion
function canFlagReportAction(reportAction: OnyxEntry<ReportAction>, reportID?: string): boolean {
```

Reply:
I believe we should leave it as it is , we want always to `reportID` to be passed but it can be `undefined`
so users of that function wont be able to call it like `canFlagReportAction(reportAction)`
WDYT?